paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/SavedPaymentMethodTabLayoutUI.kt

```diff
@@ -275,7 +275,7 @@ private fun SavedPaymentMethodTab(
                 isEnabled = isEnabled,
                 isEditing = isEditing,
                 isModifiable = item.isModifiable,
-                isRemovable = item.isRemovable,
+                isRemovable = true,
```

Comment:
Setting this to `true` for now but going to replace it in an immediate follow-up.

Reply:
should the `isRemovable` field on SavedPaymentMethodTab also be deleted? I think this was also added in the PR that added the individual remove states to DisplayableSavedPM